README.md

```diff
@@ -38,13 +38,7 @@ rake db:seed OR rake db:setup (which also sets up the entire db)
 # edit database.yml to point to your postgresql database
 cp config/database.yml.example config/database.yml
 
-cp config/app_config.yml.example config/app_config.yml
-```
-
-## Generate Secret Token
-
-```
-bundle rake secret > secret_token
+cp .env.example .env
 ```
```

Comment:
You are looking at this line. But I can add an explanation as well. The dotenv gem expects it to be named `.env` to read and load it.